json/src/de/mod.rs

```diff
@@ -98,6 +109,10 @@ where
 struct JsonDataElement<D> {
     vr: VR,
     value: Value<InMemDicomObject<D>, InMemFragment>,
+    // TODO: we just ignore this when deserializing with
```

Comment:
This will help us track the concern. From what we have discussed on Zulip, this will very likely require us to create a new object implementation.

```suggestion
    // TODO(#470): we just ignore this when deserializing with
```